doc/release-notes.md

```diff
@@ -57,6 +57,9 @@ Updated RPCs
 
 Changes to wallet related RPCs can be found in the Wallet section below.
 
+- RPC `getreceivedbylabel` now returns an error, "Label not found
+  in wallet" (-4), if the label is not in the address book. (#25122)
+
```

Comment:
nit, just saw the release note might be in the wrong section
```diff
--- a/doc/release-notes.md
+++ b/doc/release-notes.md
@@ -57,9 +57,6 @@ Updated RPCs
 
 Changes to wallet related RPCs can be found in the Wallet section below.
 
-- RPC `getreceivedbylabel` now returns an error, "Label not found
-  in wallet" (-4), if the label is not in the address book. (#25122)
-
 New RPCs
 --------
 
@@ -81,6 +78,9 @@ Tools and Utilities
 Wallet
 ------
 
+- RPC `getreceivedbylabel` now returns an error, "Label not found
+  in wallet" (-4), if the label is not in the address book. (#25122)
+
 GUI changes
 -----------
```

Reply:
pushed